addon/components/link-building-modal.js

```diff
@@ -5,7 +5,7 @@ import BuildingWork from "ember-ebau-gwr/models/building-work";
 
 // Remove the state "Neubau" from the array since you cant link
 // existing buildings as "Neubau".
```

Comment:
If I see this correctly, the comment is no longer needed after the change. So it should be removed.